test/jdk/java/util/regex/RegExTest.java

```diff
@@ -5290,4 +5291,22 @@ private static void droppedClassesWithIntersection() {
         }
 
     }
+
+    //This test is for 8269753
+    private static void errorMessageCaretIndentation() {
+        String pattern = "\t**";
+
+        try {
+            var res = Pattern.compile(pattern);
+        } catch (PatternSyntaxException e) {
+            var message = e.getMessage();
+            var sep = System.lineSeparator();
+            if(message.contains(sep + "\t ^")){
+                return;
+            }
+        }
+        failCount++;
+
+        report("Correct caret indentation for patterns with tabs");
```

Comment:
`report` will not be reached on success; this is different from how `report` is used in the rest of the file. Also: `report` seems to be stateful in that it resets the `failCount`.